k8s/sealed-secrets/ccd-aat-ccd-admin-web-oauth2-client-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  creationTimestamp: null
+  name: ccd-aat-ccd-admin-web-oauth2-client-secret
```

Comment:
I don't think you want these in the chart, default it to 123456 and then override it in the flux config for a repo

Reply:
yep agreed